_posts/2021-02-23-OpenSSL.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: Switching back to OpenSSL
+layout: post
+---
+
+The Void Linux team is planning on [performing the switch to
+OpenSSL](https://github.com/void-linux/void-packages/pull/21056) before March
+is over.
+
+For most users, this shouldn't be noticeable, since we will be pushing the same
+packages that already had to be rebuilt twice a year due to LibreSSL ABI
+breaks, but if anything in your setup depends explicitly on LibreSSL (you know
+who you are!), be sure to migrate away from that.
+
+The important parts of the post are done, but you can read on to learn about
+the decision process and some things we had to do.
+
+The discussion started in an
+[RFC](https://github.com/void-linux/void-packages/issues/20935) from April
+12th, 2020, and has been ongoing since then, with most maintainers expressing
+support for the idea. The main reasons for the switch are as listed in the RFC:
+
+- Less need for patching packaged software, which is made especially difficult
+  and potentially risky by the complexity of the OpenSSL API.
+- Better platform support for accelerated implementation on non `x86_64`
+  devices.
+- Access to new standards and algorithms earlier, such as full TLS 1.3 support.
+  Users of packages such as OpenVPN will benefit greatly from this.
+- More stable ABI, which saves builder time and user bandwidth.
+
+Since that time, Gentoo, which used to offer LibreSSL as an option, has
```

Comment:
You need a bridge paragraph here talking about Void not being the first to make this decision.  I'd talk about the fact that Alpine already dropped it, and we're reaching a point where Void is the last recognizable distro carrying libressl.

Reply:
I think Alpine dropping it was a bigger deal than Gentoo, since like Void, Alpine only offered the one option.  Gentoo always had the option of OpenSSL.  Also, Alpine dropping it had a bigger effect on Void, anyway.

Reply:
I restructured it a lot, let me know what you think.